README.md

```diff
@@ -1,2 +1,163 @@
-# pyblish-nukassist
-Pyblish for Nuke Assist
+### ![](https://cloud.githubusercontent.com/assets/2152766/6998101/5c13946c-dbcd-11e4-968b-b357b7c60a06.png)
+
+Pyblish integration for The Foundry NukeAssist.
+
+<br>
+<br>
+<br>
+
+### What is included?
+
+A set of common plug-ins and functions shared across other integrations - such as getting the current working file. It also visually integrates Pyblish into the File-menu for easy access.
+
+- Common [plug-ins](https://github.com/pyblish/pyblish-nuke/tree/master/pyblish_nuke/plugins)
```

Comment:
Some pyblish-nuke references to absolve

Reply:
Think I caught them all. Let me know if you see any more.